src/configs.py

```diff
@@ -0,0 +1,19 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
+
+
+class Settings(BaseSettings):
+    DB_USER: str
+    DB_PASSWORD: str
+    DB_HOST: str
+    DB_PORT: int
+    DB_NAME: str
+    TEST_MODE: bool
```

Comment:
If test mode is on, then testdb url is used. 
Currecntly I am using sqlite and currencly everything is compatible, but in the long run, 
I will have to change to postgres for testing too